tools/ios/MASTG-TOOL-0039.md

```diff
@@ -26,9 +26,39 @@ For a quick start you can go through the [iOS examples](https://www.frida.re/doc
 
 ## Installing Frida on iOS
 
-To connect Frida to an iOS app, you need a way to inject the Frida runtime into that app. This is easy to do on a jailbroken device: just install `frida-server` through Cydia. Once it has been installed, the Frida server will automatically run with root privileges, allowing you to easily inject code into any process.
+To connect Frida to an iOS app, you need a way to inject the Frida runtime into that app. This is easy to do on a jailbroken device since we can install `frida-server` through Sileo. Start Sileo and add Frida's repository by navigating to **Manage** -> **Sources** -> **Edit** -> **Add** and entering <https://build.frida.re>. You should then be able to find and install the Frida package.
+
+By default, `frida-server` only listens on the local interface, requiring you to connect the device over USB. If you want to expose `frida-server` on the public interface, modify `/var/jb/Library/LaunchDaemons/re.frida.server.plist` and two items to the `ProgramArguments` as shown below:
```

Comment:
Should we include a warning/danger panel for this? Exposing `frida-server` on the public interface lets anyone on the network inject code as root. It's fine for isolated lab setups, but in most cases it's safer to keep it bound to localhost and connect over USB or SSH. Might be good to highlight this so that people don’t accidentally open up a big security hole as recommended by the MASTG.